internal/storage/storageutil/test_util.go

```diff
@@ -41,5 +41,6 @@ func GetDefaultStorageClientConfig() (clientConfig StorageClientConfig) {
 		TokenUrl:                   "",
 		ReuseTokenFromUrl:          true,
 		ExperimentalEnableJsonRead: false,
+		DisableAuth:                true,
```

Comment:
Why default value is true? It should be true. Right?

Reply:
Updated